sourcegraph/jetbrains
# Use RFC 3986-compliant URIs for file paths

Fixes #218

The Java URIs uses a different standard (RFC 2396) for encoding than the library vscode-uri used by Cody agent (RFC 3986).
Due to the fact that I was unable to find a reasonable library, this patch merely imitates the encoding and normalization of such addresses.

Additional change: Remove unused CodyAgentDocuments.

## Test Plan

1. Verify basic features on Windows
2. Verify basic features on Linux
